docs/about/themes-translation.md

```diff
@@ -169,27 +133,67 @@ And run `mkdocs serve` to check out your new localized version of the theme.
     all locales to end users so you only have to worry about contributing the
     actual text translation `messages.po` files (the rest is git ignored).
 
+### Contributing your translations
+
 It's now time for you to [contribute your nice work to the project][contribute],
 thank you!
 
 [contribute]: contributing.md
 
 ## Updating theme localization files
 
-When the HTML source files of a theme change it is necessary to *extract* the
-theme text again and *update* its translated locales.
+If you changed some text in the theme themselves and modified the HTML source
+of a theme, it is necessary to follow the steps below:
+
+1. [Extract each theme's text](#extracting-text-from-themes) to update their Portable Object Template files
+2. [Update the translation catalog `messages.po` files](#updating-the-translation-catalogs) for every supported locale
+3. [Translate](#translating-the-mkdocs-themes) the newly added text placeholders on every `messages.po` file for
+every supported locale
+4. [Contribute your updated translations](#contributing-your-translations) through a Pull Request
```

Comment:
It is good that all of this is documented. But I don't expect the same person to be both editing the theme templates and updating the `po` files. Suppose we have 20 locales, each with their own `po` file. Obviously no one person would be updating all 20. Instead, the person updating the theme templates should be instructed to generate the `pot` file and include that in their PR (unless we automate it of course). Then we can inform them that the translators will see the updated `pot` file and they will provide updated translations (`po` files) separately. However, I think the directions for theme devs (those who edit theme templates) needs to go in a separate section along with some other general instructions for them (explaining that all strings in the template should be in English and wrapped in a `trans` statement, etc). I'm undecided if it should be included on this page or on the [custom theme][1] page. Either way, the two pages need to link to each other.

This section, which is directed at translators, should simply start out with something like: "If a `pot` file has been updated since the `po` was last updated for your locale, follow the steps below to update the `po` file..."

[1]: https://github.com/mkdocs/mkdocs/blob/master/docs/user-guide/custom-themes.md

Reply:
Thanks, I've split and updated the documentation as such :+1: